server/client.go

```diff
@@ -748,6 +748,15 @@ func (c *client) setPermissions(perms *Permissions) {
 			c.perms.sub.deny.Insert(sub)
 		}
 	}
+
+	// If we are a leafnode and we are the hub copy the extracted perms
+	// to resend back to soliciting server. These are reversed from the
+	// way routes interpret them since this is how the soliciting server
+	// will receive these back in an update INFO.
+	if c.isHubLeafNode() {
```

Comment:
Not sure why we are storing the perms of this client in c.opts.Import/Export? Would it not be enough to send c.perms when needed?

Reply:
This is an easy way to remember when we need to send from the hub back to the spoke via async info.